cpp/tests/CMakeLists.txt

```diff
@@ -465,6 +465,10 @@ if(BUILD_CUGRAPH_MG_TESTS)
         ###########################################################################################
         # - MG PRIMS COUNT_IF_E tests -------------------------------------------------------------
         ConfigureTestMG(MG_COUNT_IF_E_TEST prims/mg_count_if_e.cu)
+
+        ###########################################################################################
+        # - MG PRIMS COPY_V_TRANSFORM_REDUCE_OUT tests --------------------------------------------
+        ConfigureTestMG(MG_COPY_V_TRANSFORM_REDUCE_OUT_TEST prims/mg_copy_v_transform_reduce_out_nbr.cu)
```

Comment:
So, no test for in_nbr?

Reply:
And I think the name should better be ...OUT_NBR_TEST than ...OUT_TEST.

Better format this to multiple lines like

https://github.com/rapidsai/cugraph/pull/1856/files#diff-7caf2bdc6aa44337cf6e4b04392bc3e5940a7356512b4d525aba3bcda0be6c3bR436

Reply:
I think for each test, we can run both in_nbr & out_nbr, and rename test files to something like ...INOUT_NBR_TEST